test/__snapshots__/test/test.ts.md

```diff
@@ -1272,6 +1272,9 @@ Generated by [AVA](https://ava.li).
       impliedHeterogeneousEnum?: -20.1 | null | "foo" | false;␊
       namedIntegerEnumTitle: NamedInteger;␊
       impliedNamedIntegerEnumTitle: ImpliedNamedInteger;␊
+      oneOfNamedEnum: IntegerOneOfNamedEnum | StringOneOfNamedEnum;␊
+      anyOfNamedEnum: IntegerAnyOfNamedEnum | StringAnyOfNamedEnum;␊
+      allOfNamedEnum?: IntegerAllOfNamedEnum & StringAllOfNamedEnum;␊
```

Comment:
Is this right? Does re-running `yarn test -u` make `allOfNamedEnum` required?

Reply:
You have right ```allOfNamedEnum``` should be required. Good catch. 👍